nfcore_custom.config

```diff
@@ -23,6 +23,7 @@ profiles {
   crick        { includeConfig "${params.custom_config_base}/conf/crick.config" }
   czbiohub_aws { includeConfig "${params.custom_config_base}/conf/czbiohub_aws.config" }
   ebc          { includeConfig "${params.custom_config_base}/conf/ebc.config" }
+  eddie	       { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```

Comment:
```suggestion
  eddie	    { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```

Reply:
I saw that this wasn't a big enough change @MaxUlysse and edited it to remove more spaces, but I was too slow and you had already applied it 😆